### ISSUE TITLE
bias item

### ISSUE THREAD

**Issue by Yupei-Du** [asafamr/SymPatternWSI#1]
hi~ thx for sharing your code! But I still have a question: in spwsi/bilm_elmo line 110, a comment says"we prevent unconditionally probable substitutes predictions by ignoring the bias vector", what's the meaning of that? thank you a lot in advance :D

**Comment by asafamr** [comment]
intuitively, the bias term added to the softmax logits could be interpreted as a prior over the predicted words promoting the prediction words such as "the", disregarding the context and indifferent to the sense of the disambiguated word.
We empirically found out that predicting words using Softmax(Wx) instead of Softmax(Wx+b) indeed produce better substitute vectors for the sense clustering, with less probability of predicted words such as "the".
I hope this helps

**Comment by Yupei-Du** [comment]
thank a lot! it really helps me
